src/App.js

```diff
@@ -1,23 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import './components/MealCard/mealcard.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Meals from './components/Meals/Meals';
+import './components/Meals/meals.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Meals />
```

Comment:
this should be removed since in future references this will cause merge conflicts